packages/perseus/src/widgets/radio/choice-icon/choice-icon.jsx

```diff
@@ -0,0 +1,155 @@
+// @flow
+/* Component for rendering a letter icon in radio choice */
```

Comment:
**question:** Are there any changes to the files you moved into this subdirectory?

Reply:
All the little sub-components I moved over were converted to functional components with Flow instead of PropTypes where applicable.

Reply:
It would've been better to do the move and update as separate PRs.  I didn't actually look at the code in the new files before approving because I find it really to track what's changing. 😞

Reply:
Yeah, I see what you mean. 🤔 I feel like there much be some way to do a proper comparison, but maybe this is just a limitation of git? Either way I'll try to be more careful about that in the future.

Reply:
I think it's a limitation of the particular diff-ing algorithm that GitHub uses.  There are better diffing algorithms, but GitHub doesn't provide a way to configure it.